### ISSUE TITLE
authentication not working...

### ISSUE THREAD

**Issue by avnav88** [vaphes/pocketbase#14]
```import asyncio
import pocketbase
import os
import dotenv

dotenv.load_dotenv()

async def export_func(data: str | None = None):
	pb = pocketbase.PocketBase('http://localhost:8090/')
	authData = pb.admins.auth_via_email(
		email=os.environ["POCKET_ADMIN"],
		password=os.environ["POCKET_ADMIN_PASS"])
	return await authData


auth = asyncio.run(export_func())
print(auth)
```

getting `pocketbase.client.ClientResponseError: Response error. Status code:404`

in the ui:

<html>
<body>

{   "errorDetails": "<nil>",   "errorMessage": "Not Found" }
--

</body>
</html>

**Comment by avnav88** [comment]
helps very much, i wasn't aware of a new python sdk??

**Comment by dsm-72** [comment]
@ichunichu same here. I am using `0.2.2` which is the latest version of your sdk >.<

**Comment by dsm-72** [comment]
@ichunichu  see https://github.com/vaphes/pocketbase/blob/master/pocketbase/services/admins.py#L45

**Comment by dsm-72** [comment]
@ichunichu  can we get a bump on https://github.com/vaphes/pocketbase/pull/16#pullrequestreview-1248419221

**Comment by ichunichu** [comment]
The new sdk is a beta version : https://github.com/vaphes/pocketbase/releases/tag/v0.3.0-beta
It is not on pipy yet, but you still can install it

**Comment by Vrajs16** [comment]
As @ichunichu mentioned, you can compile the source code using python poetry by downloading the tar and then running the following inside the directory until it is published to Pypi:
```bash
# Install poetry on mac
$ brew install poetry

# Install dependencies
$ poetry install

# Build package
$ poetry build

# Add pocketbase to pip
$ pip install dist/pocketbase*.whl 
```
And then your good to go!

**Comment by m29h** [comment]
closing this issue as it is long solved in the [current release v0.8.2](https://pypi.org/project/pocketbase/)
